netbox/netbox/views/generic/object_views.py

```diff
@@ -327,6 +327,12 @@ def alter_object(self, obj, request, url_args, url_kwargs):
         """
         return obj
 
+    def get_extra_addanother_params(self, request, params):
```

Comment:
Need to drop `params` from the default signature.

```suggestion
    def get_extra_addanother_params(self, request):
```

Reply:
doh! should have caught that.